multilabel_guide.md

```diff
@@ -0,0 +1,49 @@
+# **Multilabel classification tutorial**
```

Comment:
I propose to write the title of the paper instead of "Multilabel classification tutorial"
and we need to add a link to paper as well as the reference to papers with code (after we release the paper)